graylog2-server/src/main/java/org/graylog2/rest/resources/entities/preferences/EntityListPreferencesResource.java

```diff
@@ -63,7 +63,7 @@ public EntityListPreferencesResource(final EntityListPreferencesService entityLi
     @Timed
     @ApiOperation(value = "Create or update user preferences for certain entity list")
     @Consumes(MediaType.APPLICATION_JSON)
-    @NoAuditEvent("Storage details have not yet been decided") //TODO
+    @NoAuditEvent("Audit logs are not stored for entity list preferences") //TODO: make sure it is true
```

Comment:
Should we resolve this TODO before merging?

Reply:
_TLDR - I do not see any big risk, but we can wait till the evening to merge after release, to be on the safe side._

I personally think that we should not store audit log every time a user changes visible columns or page size in any entity list. I'd love to consult it with Kay (he is back in a few days). If he had different opinion, I'll simply add it.
For now nobody uses this endpoints, there is no FE ready. 
It would be nice to have it merged so that FE works on the final solution when developing FE part.
I suggest merging after release and quickly adding audit logging if needed.